code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -720,3 +720,33 @@
 	storage_slots = 10
 	can_only_hold = list("/obj/item/weapon/spacecash", "/obj/item/weapon/coin")
 
+/*
+ * Beer Box
+ */
+
+/obj/item/weapon/storage/fancy/beer_box
+	icon = 'icons/obj/food_container.dmi'
+	icon_state = "beerbox6"
+	icon_type = "beer"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/boxes_and_storage.dmi', "right_hand" = 'icons/mob/in-hand/right/boxes_and_storage.dmi')
+	item_state = "beerbox"
+	name = "beer box"
+	storage_slots = 6
+	can_only_hold = list("/obj/item/weapon/reagent_containers/food/drinks/beer")
+
+	foldable = /obj/item/stack/sheet/cardboard
+	starting_materials = list(MAT_CARDBOARD = 3750)
+	w_type = RECYK_MISC
+
+/obj/item/weapon/storage/fancy/beer_box/empty
+	empty = 1
+	icon_state = "beerbox0"
+
+/obj/item/weapon/storage/fancy/beer_box/New()
+	..()
+	if(empty)
+		update_icon()
+		return
+	for(var/i = 1; i <= storage_slots; i++)
```

Comment:
```suggestion
	for(var/i in 1 to storage_slots)
```
it's preferred to use this kind of for-loops. they are easier to read and faster (not that it matters here).